src/Controller/EnMarche/Coordinator/CoordinatorCitizenProjectController.php

```diff
@@ -0,0 +1,91 @@
+{
+    /**
+     * @Route(path="/list", name="app_coordinator_citizen_project")
+     */
+    public function listAction(Request $request)
+    {
+        try {
+            $filters = CitizenProjectFilter::fromQueryString($request);
+        } catch (\UnexpectedValueException $e) {
+            throw new BadRequestHttpException('Unexpected Citizen Project status in the query string.', $e);
+        }
+
+        $results = $this->get('app.citizen_project.manager')->getCoordinatorCitizenProjects($this->getUser(), $filters);
+
+        $forms = [];
+        array_walk($results, function (CitizenProject $project) use (&$forms, $filters) {
+            $forms[$project->getId()] = $this
+                ->createForm(CoordinatorAreaType::class, $project, [
+                    'data_class' => CitizenProject::class,
+                    'action' => $this->generateUrl('app_coordinator_citizen_project_validate', [
+                        'uuid' => $project->getUuid(),
+                        'slug' => $project->getSlug(),
+                    ]),
+                    'status' => $filters->getStatus(),
+                ])
+                ->createView();
+        });
+
+        return $this->render('coordinator/citizen_project.html.twig', [
+            'results' => $results,
+            'filters' => $filters,
+            'forms' => $forms,
+        ]);
+    }
+
+    /**
+     * @Route("/{uuid}/{slug}/pre-valider", name="app_coordinator_citizen_project_validate")
+     * @Method("POST")
+     */
+    public function validateAction(Request $request, CitizenProject $project): Response
+    {
+        $form = $this
+            ->createForm(CoordinatorAreaType::class, $project, [
+                'data_class' => CitizenProject::class,
+            ])
+            ->handleRequest($request);
+
+        if ($form->isSubmitted() && $form->isValid()) {
+            try {
+                if ($form->get('refuse')->isClicked()) {
+                    $this->get('app.citizen_project.authority')->preRefuse($project);
+                    $this->addFlash('info', 'Merci. Votre appréciation a été transmise à nos équipes.');
+                }
+
+                if ($form->get('accept')->isClicked()) {
+                    $this->get('app.citizen_project.authority')->preApprove($project);
+                    $this->addFlash('info', 'Merci. Votre appréciation a été transmise à nos équipes.');
+                }
+            } catch (BaseGroupException $exception) {
+                throw $this->createNotFoundException(sprintf('Citizen project %u has already been treated by an administrator.', $project->getId()), $exception);
+            }
+        } else {
+            foreach ($form->getErrors(true) as $error) {
+                $this->addFlash('error_'.$form->getData()->getId(), $error->getMessage());
+            }
+        }
+
+        return $this->redirectToRoute('app_coordinator_citizen_project', ['s' => CitizenProject::PENDING]);
```

Comment:
For `app_coordinator_citizen_project ` route, there is no `s` parameter ?

Reply:
`s` (for _status_) parameter is used for filter purposes;
see https://github.com/EnMarche/en-marche.fr/pull/1825/files#diff-732435785991ed33606c03f60da392d7R29
& https://github.com/EnMarche/en-marche.fr/pull/1825/files#diff-ac23d0cd9617d08e52f5b3dd2f50fc8aR59

Reply:
@tony-tran yes, it's a query param (GET) and not of route. 
@Remg fixed 👍